app/services/redacting/unredact.rb

```diff
@@ -0,0 +1,36 @@
+module Redacting
```

Comment:
Hmm on second thought I believe it may be more robust the way I'm doing it now, as "Redacting the main payload and checking it against the already redacted one to ensure they are identical" will always be found to be true, but doesn't guarantee we are doing a proper redacting of only the attributes we care, and leaving the other attributes intact, etc.

I'm inclined to leave this Unredact class as a utility class if only for the tests. Is not a lot of code anyways?

Reply:
I've added an explanatory comment in the code as well as make sure we compare the whole payload in the spec, hopefully making this more clear.